trac-env/htdocs/tickethacks.js

```diff
@@ -229,7 +229,7 @@ $(function() {
         );
     } else if (stage == 'Someday/Maybe') {
         next_steps.push(
-            '<p>Unknown. The Someday/Maybe triage stage is used by core developers ' +
+            '<p>Unknown. The Someday/Maybe triage stage is used by triagers ' +
```

Comment:
I would remove `by triagers`
```suggestion
            '<p>Unknown. The Someday/Maybe triage stage is used ' +
```